test/TesterInternal/General/RequestContextTest.cs

```diff
@@ -169,22 +181,33 @@ public void Dispose()
 
             IRequestContextTestGrain grain = this.fixture.GrainFactory.GetGrain<IRequestContextTestGrain>(GetRandomGrainId());
 
+#if NETSTANDARD
+            RequestContext.ActivityId.Value = activityId;
+#else
             Trace.CorrelationManager.ActivityId = activityId;
+#endif
            Assert.Null(RequestContext.Get(RequestContext.E2_E_TRACING_ACTIVITY_ID_HEADER));
             Guid result = await grain.E2EActivityId();
             Assert.Equal(activityId,  result);  // "E2E ActivityId not propagated correctly"
             RequestContext.Clear();
 
+#if NETSTANDARD
+            RequestContext.ActivityId.Value = nullActivityId;
```

Comment:
Although agree with the original pain point. I think our tests can use a wrapper that just has a NETSTANDARD_TODO wrapper in front of it, so we don't need to have conditional compilation ALL over the place, just in 1 place in our test project.